daemon/nextbox-compose/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '2'
+version: '2.1'
```

Comment:
uh, this gives me paranoia feelings - this is a deep-seated pain, especially with regard to the possibility of sharing an externally mounted hard disk within /media for nextcloud ... please test explicitly: attach an external storage through the nextbox-app, then attach this one via the "External Storage" nextcloud app to be available inside the "Files" app in Nextcloud